setup.py

```diff
@@ -95,6 +95,7 @@ def about_info(package: str):
             "captum~=0.2.0",
             "ipywidgets~=7.5.1",
             "mlflow~=1.9.0",
+            "ray>=0.8.6,<0.9.0"
```

Comment:
they already released 1.0, can we use that?

Reply:
done